README.md

```diff
@@ -1 +1,56 @@
-# covid-tracking
\ No newline at end of file
+# Covid Tracking
+
+Web application that consumes an open source api about covid stadisctics around the world, with this application you can be informed about the covid status for almost every country.
+
```

Comment:
### Live version

- [ ] Deploy the application using Heroku, Netlify, or GitHub Pages.

- [ ] The project is deployed properly, the home and detail page works as expected.

- [ ] A link to the deployed project is present in the README.